index.js

```diff
@@ -72,13 +72,14 @@ function beautify(error, collection, values, messages) {
                 indexes[indexName].forEach(function (field) {
                     var path = field[0];
                     var props = {
-                        type: 'Duplicate value',
+                        type: 'unique',
+                        kind: 'unique',
                         path: path,
                         value: values[path]
                     };
 
                     if (typeof messages[path] === 'string') {
-                        props.message = messages[path];
+                        props.message = props.reason = messages[path];
```

Comment:
I can get rid of it - just let me know about my other comment above <https://github.com/matteodelabre/mongoose-beautiful-unique-validation/pull/40#discussion_r131781824> and I'll do it all at once.